chatwoot/chatwoot
# chore: Execute campaigns based on matching url

- Start campaigns based on the URL specified in trigger rules  

## How to Test
1. Enable inbox campaigns

Open `app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue`
Add the code on line number 337

```
{
 key: 'campaign',
 name: this.$t('INBOX_MGMT.TABS.CAMPAIGN'),
}
```
2. Go to any website inbox

3. Click on the settings

4. Go to the campaigns tab

5. Create a new campaign by entering details title, messages, sender, URL and time on page

6. Open the URL which configured website inbox

7. Wait for the time specified in the campaign

8. Check messages are coming over the widget
